tests/contrib/molten/test_molten.py

```diff
@@ -0,0 +1,48 @@
+from unittest import TestCase
+
+import molten
+from ddtrace import Pin
+from ddtrace.contrib.molten import patch
+from molten import App, Route
+from molten.testing import TestClient
+from nose.tools import eq_, ok_
+
+from ...test_tracer import get_dummy_tracer
+
+
+class TestMolten(TestCase):
+    """"Ensures Molten is properly instrumented."""
+
+    TEST_SERVICE = 'molten-patch'
+
+    def setUp(self):
+        self.tracer = get_dummy_tracer()
+        # simple hello app
+        def hello(name: str, age: int) -> str:
+            return f"Hello {age} year old named {name}!"
+        self.app = App(routes=[Route("/hello/{name}/{age}", hello)])
+        patch()
+
+    def tearDown(self):
+        pass
```

Comment:
We should probably do:

```python
delattr(self, 'tracer')
delattr(self, 'app')
```

although you are overwriting in `setUp` we want to make sure those instances don't leak outside of these test cases.